arviz/tests/external_tests/test_data_pystan.py

```diff
@@ -1,19 +1,31 @@
 # pylint: disable=no-member, invalid-name, redefined-outer-name
+import importlib
 from collections import OrderedDict
+
 import numpy as np
 import pytest
 
 from arviz import from_pystan
+
 from ...data.io_pystan import get_draws, get_draws_stan3  # pylint: disable=unused-import
 from ..helpers import (  # pylint: disable=unused-import
     chains,
     check_multiple_attrs,
     draws,
     eight_schools_params,
+    importorskip,
     load_cached_models,
     pystan_version,
 )
 
+# Check if either stan or pystan is installed
+pystan_installed = (importlib.util.find_spec("pystan") is not None) or (
+    importlib.util.find_spec("stan") is not None
+)
+pytestmark = pytest.mark.skipif(
+    ~pystan_installed, reason="test requires stan/pystan which is not installed"
```

Comment:
Maybe this could be

`... requires pystan/pystan3 which is not ...`

Reply:
Sure I'll make that change